### ISSUE TITLE
Unable to import stow due to invalid syntax

### ISSUE THREAD

**Issue by stephane-archer** [caian-org/ansible-stow#3]
the following script result in the following error
```
#!/bin/bash
wget -O stow https://git.io/fjlZ4
sudo mkdir -p /usr/share/ansible/plugins/modules/
sudo mv stow /usr/share/ansible/plugins/modules/
```
```
fatal: [localhost]: FAILED! => {"msg": "Unable to import stow due to invalid syntax"}
```
with the following version
```
ansible-playbook 2.9.27
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/steph/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 2.7.17 (default, Apr 15 2020, 17:20:14) [GCC 7.5.0]
```

**Comment by upsetbit** [comment]
Hello @stephane-archer.

I've noticed that your ansible installation is using Python 2.7; that's probably the reason for this error.

Somewhat recently I've changed a couple of things on this project. This commit (https://github.com/caian-org/ansible-stow/commit/ead1099b22cb61c82ff7ac6a8c36f2d14d92a18d) for instance replaced `.format` for f-strings -- and this feature is only available on versions `>= 3.6`.

So:
- You can upgrade your python version to be `>= 3.6`
- Or use and older version of `ansible-stow`

This should work:

```sh
#!/bin/bash
wget -O stow https://raw.githubusercontent.com/caian-org/ansible-stow/v0.1.3/stow
sudo mkdir -p /usr/share/ansible/plugins/modules/
sudo mv stow /usr/share/ansible/plugins/modules/
```

I've updated the README to include info about the required python version; this way, the next person hopefully would not get the same problem as you.

**Comment by upsetbit** [comment]
How are you, @stephane-archer? It's been a while since your first message. I'm closing this issue but feel free to reopen if you're still facing problems.
